server/db/index.test.js

```diff
@@ -1,16 +1,53 @@
-const db = require('./index');
+require('dotenv').config();
 
+jest.mock('pg');
 
-describe('db/index.js', () => {
-  xtest('should connect to the database', () => {
+const db = require('./index');
 
+describe('db/index.js', () => {
+  describe('getOpenSeats', () => {
+    test('should expose method "getOpenSeats"', () => {
+      expect(db.getOpenSeats).toBeInstanceOf(Function);
+    });
   });
 
-  test('should expose method "getOpenSeats"', () => {
-    expect(db.getOpenSeats).toBeInstanceOf(Function);
+  describe('addRestaurantInfo', () => {
+    test('should run insert query of restaurant info into DB', () => {
+      const testItem = { id: 5, name: 'Krusty Burger', seats: 100 };
+      db.addRestaurantInfo(testItem)
+        .then((result) => {
+          expect(result.query).toBe('INSERT INTO restaurants (id,name,seats) VALUES ($1,$2,$3)');
+          expect(result.params).toHaveLength(3);
+          expect(result.params).toEqual(expect.arrayContaining([
+            testItem.id, testItem.name, testItem.seats]));
+        })
+        .catch((error) => {
+          expect(error).toBe(false);
+        });
+    });
   });
 
-  test('should expose method "addReservation"', () => {
-    expect(db.addReservation).toBeInstanceOf(Function);
+
+  describe('addReservation', () => {
+    test('should run insert query of reservation into DB', () => {
+      const testItem = {
+        restaurantId: 1,
+        date: '2018-03-01',
+        time: 17,
```

Comment:
I should've asked previously in your fake data commit, but how does t in time : t translate in time? is it just minus 12

Reply:
No problem.  Not sure if I specified it anywhere.  Time is 24H format, so 5pm is 17, 6pm is 18, etc.